src/Sylius/Bundle/MailerBundle/Sender/Adapter/SwiftMailerAdapter.php

```diff
@@ -49,8 +52,12 @@ public function send(array $recipients, $senderAddress, $senderName, RenderedEma
 
         $message->setBody($renderedEmail->getBody(), 'text/html');
 
+        $emailSendEvent = new EmailSendEvent($message, $recipients, $email, $data);
```

Comment:
The ``RenderedEmail`` is passed in the ``EMAIL_RENDERED`` event, so if you need to change something on it you can listen to that event.
But if you think it should be added I can do it.
I'm using also ``EmailInterface`` as it contains information such as ``code``